app/views/admin/submissions/show.html.erb

```diff
@@ -60,7 +60,11 @@
             </div>
             <div class='bold-label single-padding-top'>Assigned To:</div>
             <div class='single-padding-top'>
-              <%= @submission.assigned_to || 'Unassigned' %>
+              <%= form_with model: [:admin, @submission] do |f| %>
```

Comment:
This is cool, I didn't know it would be so easy to add an "inline action" to the page. 

One thing I can't figure out, and it might be a really dumb question: what does `:admin` refer to in the `model`?

Reply:
Yeah I wasn't expecting this to be easy but then I remembered that our pal @erikdstock added that notes section to this page with https://github.com/artsy/convection/pull/707 and so I looked at how he did it and it turned out I could copy him! His approach was to add a form to the page that posts to a Rails controller and then has that Rail controller redirect back to the submission show page. Since the submission has been updated prior to that second rendering, the page will be up-to-date.

This is not as cool as making a request with JS and then updating the DOM and having no page refresh, but that seemed like an acceptable tradeoff because this is an admin app.

Now to your question: that `form_with mode: [:admin, @submission]` stuff is a DSL for creating a form tag. The reason to send in an array of admin and submission is so that the form tag builder spits out a POST that has admin prefixed:

```ruby
# this route exists!
form_with model: [:admin, @submission] # => POST to /admin/submissions/:id
# this is a 404
form_with model: @submission # => POST to /submissions/:id
```

Reply:
oh, I think I have a misconception about how it builds the form submission route. 

I was thinking it would take the model type (submission) and build a route from that, relative to where you currently were in the app. It sounds like it takes the model type and builds a route from that, relative to the root, unless you tell it otherwise. Do I have that right?

Reply:
"I could copy him!" - the easiest code ever written.

Reply:
Yeah it sounds like you're about there - if you want to read more, I would recommend this section:

https://guides.rubyonrails.org/form_helpers.html#binding-a-form-to-an-object

FWIW, I never get this stuff right on the first try and usually tinker with it and read docs until I have the magic incantation correct and then I pretty much never have to change it. 🤷

Reply:
yes, i was surprised to learn about this as well... although if I remember correctly yuki pointed out there might be other, more semantic ways to do it but this served me well.